src/main/java/uk/gov/hmcts/reform/translate/controllers/DictionaryController.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.translate.controllers;
+
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.translate.model.GetDictionaryResponse;
+import uk.gov.hmcts.reform.translate.service.DictionaryService;
+
+@RestController
+public class DictionaryController {
+
+    private final DictionaryService dictionaryService;
+
+    @Autowired
+    public DictionaryController(DictionaryService dictionaryService) {
+        this.dictionaryService = dictionaryService;
+    }
+
+    @GetMapping(path = "/dictionary", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
Apparently GET / PUT and other services are going to use as unique resource /dictionary. Then the hppt verbs GET PUT PATH etc are going to identify the  restful resource. Could we move /dictionary to the to of the controller in order to following the design and avoid duplications .

Reply:
will fix